expyfun/stimuli/_crm.py

```diff
@@ -0,0 +1,487 @@
+# -*- coding: utf-8 -*-
+"""Functions for using the Coordinate Response Measure (CRM) corpus.
+"""
+
+import os
+from os.path import join
+import numpy as np
+from mne.filter import resample
+import expyfun.visual as vis
+from ._stimuli import window_edges
+from ..io import read_wav, write_wav
+from .._utils import fetch_data_file, _get_user_home_path
+from zipfile import ZipFile
+from joblib import Parallel, delayed, cpu_count
+import struct
+
+#from scipy.signal import resample
+
+_fs_binary = 40e3  # the sampling rate of the original corpus binaries
+_rms_binary = 0.099977227591239365  # the RMS of the original corpus binaries
+_rms_prepped = 0.01  # the RMS for preparation of the whole corpus at an fs
+
+_sexes = {
+    'male': 0,
+    'female': 1,
+    'm': 0,
+    'f': 1,
+    0: 0,
+    1: 1}
+_talker_nums = {
+    '0': 0,
+    '1': 1,
+    '2': 2,
+    '3': 3,
+    0: 0,
+    1: 1,
+    2: 2,
+    3: 3}
+_callsigns = {
+    'charlie': 0,
+    'ringo': 1,
+    'laker': 2,
+    'hopper': 3,
+    'arrow': 4,
+    'tiger': 5,
+    'eagle': 6,
+    'baron': 7,
+    'c': 0,
+    'r': 1,
+    'l': 2,
+    'h': 3,
+    'a': 4,
+    't': 5,
+    'e': 6,
+    'b': 7,
+    0: 0,
+    1: 1,
+    2: 2,
+    3: 3,
+    4: 4,
+    5: 5,
+    6: 6,
+    7: 7}
+_colors = {
+    'blue': 0,
+    'red': 1,
+    'white': 2,
+    'green': 3,
+    'b': 0,
+    'r': 1,
+    'w': 2,
+    'g': 3,
+    0: 0,
+    1: 1,
+    2: 2,
+    3: 3}
+_numbers = {
+    'one': 0,
+    'two': 1,
+    'three': 2,
+    'four': 3,
+    'five': 4,
+    'six': 5,
+    'seven': 6,
+    'eight': 7,
+    '1': 0,
+    '2': 1,
+    '3': 2,
+    '4': 3,
+    '5': 4,
+    '6': 5,
+    '7': 6,
+    '8': 7,
+    0: 0,
+    1: 1,
+    2: 2,
+    3: 3,
+    4: 4,
+    5: 5,
+    6: 6,
+    7: 7}
```

Comment:
Yeah, there's a warning about it elsewhere in the code. Basically, I wanted to give people access to the corpus via the "name" of each parameter ('charlie', '1'), or by its index in the list of options. And the index of '1' is 0.